nano/node/nodeconfig.cpp

```diff
@@ -473,6 +476,10 @@ nano::error nano::node_config::deserialize_toml (nano::tomlconfig & toml)
 		{
 			toml.get_error ().set ("active_elections_size must be greater than 250");
 		}
+		if (max_unchecked_blocks < 1)
```

Comment:
There is practically no difference between 1 and 0, so if someone wants to disable unchecked blocks let's allow it. It might be useful for testing.

Reply:
Good point. Updated